frontend/src/components/messstelle/optionsmenue/composable/belastungsplanAnzeigeUtils.ts

```diff
@@ -1,64 +1,65 @@
 import { computed } from "vue";
+
 import { useMessstelleStore } from "@/store/messstelle";
 
 export function belastungsplanAnzeigeUtils() {
-    const messstelleStore = useMessstelleStore();
+  const messstelleStore = useMessstelleStore();
 
-    const chosenOptionsCopy = computed(() => {
-        return messstelleStore.getFilteroptions;
-    });
+  const chosenOptionsCopy = computed(() => {
+    return messstelleStore.getFilteroptions;
+  });
 
-    const chosenOptionsCopyFahrzeuge = computed(() => {
-        return chosenOptionsCopy.value.fahrzeuge;
-    });
+  const chosenOptionsCopyFahrzeuge = computed(() => {
+    return chosenOptionsCopy.value.fahrzeuge;
+  });
 
-    const isSv_pInBelastungsPlan = computed(() => {
-        let actualNumberOfSelectedKfzSvAndGv = 0;
-        if (chosenOptionsCopyFahrzeuge.value.kraftfahrzeugverkehr) {
-            actualNumberOfSelectedKfzSvAndGv++;
-        }
-        if (chosenOptionsCopyFahrzeuge.value.schwerverkehr) {
-            actualNumberOfSelectedKfzSvAndGv++;
-        }
-        if (chosenOptionsCopyFahrzeuge.value.gueterverkehr) {
-            actualNumberOfSelectedKfzSvAndGv++;
-        }
-        return (
-            chosenOptionsCopyFahrzeuge.value.schwerverkehrsanteilProzent &&
-            (chosenOptionsCopyFahrzeuge.value.kraftfahrzeugverkehr ||
-                chosenOptionsCopyFahrzeuge.value.schwerverkehr ||
-                chosenOptionsCopyFahrzeuge.value.gueterverkehr) &&
-            actualNumberOfSelectedKfzSvAndGv < 3
-        );
-    });
+  const isSv_pInBelastungsPlan = computed(() => {
```

Comment:
Schreibweise in Camelcase.

`isSv_pInBelastungsPlan` -> `isSvpInBelastungsPlan`